udata/core/badges/metrics.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from udata.core.metrics import Metric
+from udata.i18n import lazy_gettext as _
+
+from .signals import on_badge_added, on_badge_removed
+
+
+class BadgesMetric(Metric):
+    name = 'badges'
+    display_name = _('Badges')
+
+    def get_value(self):
+        return (self.model_badge.objects(subject=self.target, removed=None)
```

Comment:
Actually, `self.model` is the subject, see https://github.com/etalab/udata/pull/55/files#diff-8b1584d4d7d89f813bfbf43533ce51dfR50 for instance and `model_badge` is the inherited model from `Badge`.